packages/camera/camera/example/pubspec.yaml

```diff
@@ -8,12 +8,9 @@ environment:
 
 dependencies:
   camera:
-    # When depending on this package from a real application you should use:
```

Comment:
We should keep this for users that read our example application code.

Reply:
Restored comment.